admin/client.py

```diff
@@ -44,7 +41,7 @@ class DockerImage(object):
 }
 
 PIP_DISTRIBUTIONS = DOCKER_IMAGES.keys()
-PACKAGED_CLIENT_DISTRIBUTIONS = ('ubuntu-14.04', 'ubuntu-15.04')
+PACKAGED_CLIENT_DISTRIBUTIONS = ('centos-7', 'ubuntu-14.04', 'ubuntu-15.04')
```

Comment:
CentOS is supported here, because even though it is an unsupported platform, the client packages are created, and installing them is the only way we check the RPM packages.  This is explained in `test-artifacts`, but I will include a comment here as well.